pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: country_codes
 description: Country codes is an helper package that provides country details given a particular localization, such as dial codes, ISO 3166 codes (alpha-2, alpha-3) and country names.
-version: 1.0.3
+version: 1.1.0
```

Comment:
This should be bumped to major (`2.0.0`).

Reply:
done